.travis.yml

```diff
@@ -2,24 +2,39 @@ sudo: required
 dist: trusty
 language: node_js
 node_js:
-- '8'
+  - '10'
+before_install:
+  - npm i -g npm@5.3.0
+  - gem install bundler -v 1.16.1
+install:
+  - bundle install
+  - npm install
+before_script:
+  - npm run build
+script:
+  - npm run test:e2e
 env:
   global:
-  - CC_TEST_REPORTER_ID: 6f5024bb1ebb518893991cf5d77e75e4819eaa6128fe21bf85ef3bf135cd49d2
-  - BROWSER_STACK_USERNAME: bugsnagplatforms1
-  - secure: "X29QA8LjTZL9RHnoiYlprpIbv264e//KjL3Q4A/vWZ8QbIRD2MY8w9wuIfQEYHLGa7kf1EAGPoH/Ka3raj7s5+J4hOeCeMC0ZWUWdAem35FE7zGm8wJrCKFK5GD9LF+1LRxK+gBfKfeaJkwcbjv7tl0DEW1JNC9PS4cpI0DRJzM="
+    - BROWSERSTACK_USERNAME: bugsnagplatforms1
+    - secure: "tt/Rk4trBHP2+776EQPmpInHXGWojZYGLpvm2KPjw6Wb+PzK+bJiVoox2PZ1tfKl834CxpPa8yJK26Bh1/qgxf0cNOC9cY2QyLJCIOCVAMPQ0jhNj4l9ouOe+MfA0cHpOLb91rKtRcwF6YSZ5sHVG3bF1Jk51Q8drMmTqj/GHYk="
+  matrix:
+    - BROWSER=ie_8
+    - BROWSER=ie_9
+    - BROWSER=ie_10
+    - BROWSER=ie_11
+    - BROWSER=edge_14
+    - BROWSER=edge_15
+    - BROWSER=safari_6
+    - BROWSER=safari_10
+    - BROWSER=opera_12
+    - BROWSER=iphone_7
+    - BROWSER=android_s8
+    - BROWSER=firefox_30
+    - BROWSER=firefox_56
+    - BROWSER=chrome_43
+    - BROWSER=chrome_61
 addons:
-  chrome: stable
   browserstack:
     username: bugsnagplatforms1
     access_key:
      secure: "lB2aZc0HzuinyhSJyP98lzDJTbZsroEOJTrS/flnLLsHb/C096zjhwkWexHnfmNIPaY5SB2FF0G6zcnknlmsXg5TgxQiK7bdyJmkcAetGi+cSKnGaBq7GZXVhbWh++MRnddQDGV6jrSr7aqrWf85ncq3L4//9BTV6uAhDBWu2bo="
-before_script:
-- curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64
```

Comment:
Has this been replaced elsewhere?

Reply:
No, it needs to be readded as per the "changed" section in the PR body (those two sets of tests are what generate the coverage).